src/Fluidity/Configuration/FluidityCollectionConfig`T.cs

```diff
@@ -342,7 +375,12 @@ public new FluidityListViewConfig<TEntityType> ListView(Action<FluidityListViewC
         /// <returns>The list view configuration.</returns>
         public new FluidityListViewConfig<TEntityType> ListView(FluidityListViewConfig<TEntityType> listViewConfig)
         {
-            _listView = listViewConfig;
+            if (_canDelete)
+            {
+                listViewConfig.AddBulkAction<FluidityDeleteBulkAction>();
```

Comment:
Hmm,by moving this here, the delete bulk action is now at the front of the bulk actions list, where as before, it was appended on the end. I think this might need tweaking so that the delete bulk action always stays at the end.

Reply:
Good shout, but from testing this isn't the case. The ListViewConfig gets passed in as the user has configured it, and then the delete bulk action gets added if the collection allows deletion. Moved the call to here since the ListViewConfig doesn't appear to have any reference as to what the collection can do.

Reply:
Don't forget, passing a config is only one way of configuring a list view. So if you pass the whole config, that's fine, but you can also do:

````
collectionConfig.ListView().AddBulkAction<MyBulkAction>()
````

So in this instance, ListView wasn't passed anything, so it created a blank config, calling your implementation, adding the delete action. However it then returns the config for further fluent confguration, which I then add another bulk action, which is now at the end, and the delete action is now in front.

Reply:
Ok that makes sense and I missed that option. How would you get around that the list view doesn't appear to be aware what the collection can do? In the previous scenario, the delete bulk action was always added regardless.

Reply:
Maybe reinstate `_defaultBulkActions` and when _canDelete is changed, add / remove the bulk action from the _defaultBulkActions collection, but then keep the concatination we had before. So it's always appended when the final list is requested.

Might need to be carefull if we set _canDelete from within other config methods. If we do, it might be worth changing those so that they call the public api, rather than setting _canDelete explicitly. That way you can encapsulate all the add / remove logic in the DisableDelete method.

Reply:
Ok thanks, will take a look at that approach.